index.html

```diff
@@ -1412,6 +1412,128 @@ <h3>Biometrics</h3>
+        should be taken when securing an identifier system that supports these critical 
+        activities. As with all things, strong consideration for the appropriate and ethical 
+        use of technology should be made when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">Principles</a>
+        section of the Ethical Web Principles, there are certain key goals that should 
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Energy Usage and Environmental Impacts</h3>
+      <p>
+        When implementing or utilizing a DID method, consideration should be given
+        to the environmental impacts of any underlying technologies.  
+        Avoidance of excess energy use that does not further other equally important 
+        goals such as human rights or personal privacy, particularly when that energy 
+        is sourced from non-sustainable energy creation methods, should be a best 
+        practice that serves the interest of all.
+      </p>
+      <p>
+        Direct comparison of each factor, and the balances between them, should be
+        considered when evaluating particular methods or approaches. Notably, the
+        ability to misuse or control DIDs that could impede privacy might be
+        found to be inversely correlated with additional energy consumption.
+      </p>
+      <p>
+        The guiding principle that 
+        <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+        web must be an environmentally sustainable platform</a> should be followed.
+      </p>
+
+      <p class="advisement">
+        Utilizing or authoring DID methods that require unsustainably-sourced 
+        energy as part of their technical implementation or utilization should
+        be very carefully considered in balance against alternative approaches.
+      </p>
+
+      <p>
+        When selecting a DID method, the method that minimizes energy consumption 
+        while meeting privacy, interoperability, and other technical requirements
+        should generally be chosen.
+      </p>
+
+      <p class="advisement">
+        It is strongly recommended that authors of DID methods provide an assessment
+        of energy usage and impact of their DID method, preferably performed by
+        an independent third party.
+      </p>
```

Comment:
This is really outside the bounds of what a protocol implementer should be asked to do by a developer guide. Additionally, every DID method imaginable (besides self-resolving ones like did:key) will change in consumption and behavior over time, so to ask people to do running third-party energy assessments is particularly strange.

To illustrate how strange this is, imagine you asked everyone using BitTorrent as the basis for file transfer to do an environmental report on all BitTorrent servers, traffic, and resource usage on machines across the world. We should not be injecting ourselves into the use of systems like this, imo.